validmind/test_suites/classifier.py

```diff
@@ -24,8 +24,7 @@ class ClassifierMetrics(TestSuite):
         "validmind.model_validation.ModelMetadata",
         "validmind.data_validation.DatasetSplit",
         "validmind.model_validation.sklearn.ConfusionMatrix",
-        "validmind.model_validation.sklearn.ClassifierInSamplePerformance",
-        "validmind.model_validation.sklearn.ClassifierOutOfSamplePerformance",
+        "validmind.model_validation.sklearn.ClassifierPerformance",
```

Comment:
Could you create a PR on the backend so we can update the templates that rely on this?

Reply:
I have created a separate PR in backend repo.